misc/python/materialize/mzcompose/services/sql_logic_test.py

```diff
@@ -14,20 +14,20 @@
 from materialize.mzcompose.service import (
     Service,
 )
-from materialize.mzcompose.services.postgres import METADATA_STORE
 
 
 class SqlLogicTest(Service):
     def __init__(
         self,
         name: str = "sqllogictest",
         mzbuild: str = "sqllogictest",
-        environment: list[str] = [
-            "MZ_SOFT_ASSERTIONS=1",
-        ],
+        environment: list[str] | None = None,
         volumes: list[str] = ["../..:/workdir"],
-        depends_on: list[str] = [METADATA_STORE],
     ) -> None:
+        if not environment:
```

Comment:
Shouldn't this be `if environment is not None:`? Otherwise this will also evaluate as true when you pass an empty list.